erpnext/accounts/report/customer_ledger_summary/customer_ledger_summary.py

```
@@ -306,12 +306,15 @@ def get_data(self):
 				or row.return_amount
 				or row.closing_balance  # Fixed typo from closing_amount to closing_balance
 			):
+				adjustments = self.party_adjustment_details.get(party, {})
+				# Fix double counting by subtracting already-allocated advances
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)